components/clarity-repl/src/repl/session.rs

```diff
@@ -697,8 +700,16 @@ impl Session {
         }
     }
 
-    pub fn lookup_api_reference(&self, keyword: &str) -> Option<&String> {
-        self.api_reference.get(keyword)
+    pub fn lookup_functions_or_keywords_docs(&self, exp: &str) -> Option<&String> {
+        if let Some(function_doc) = self.api_reference.get(exp) {
+            return Some(function_doc);
+        } else {
+            if let Some(keyword_doc) = self.keywords_reference.get(exp) {
+                return Some(keyword_doc);
+            } else {
+                return None;
+            }
+        }
```

Comment:
```suggestion
        if let Some(function_doc) = self.api_reference.get(exp) {
            return Some(function_doc);
        }
        if let Some(keyword_doc) = self.keywords_reference.get(exp) {
            return Some(keyword_doc);
        }
        None
```

Because of the early returns, the `else` are not needed here. Helps to avoid some nesting

Reply:
@hugocaillard requested changes done. Thanks!